pkg/instrumentation/kubernetes_api_client.go

```diff
@@ -0,0 +1,49 @@
+package instrumentation
+
+import (
+	"github.com/grafana/dskit/instrument"
+	"github.com/opentracing-contrib/go-stdlib/nethttp"
+	"github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"k8s.io/client-go/rest"
+	"net/http"
```

Comment:
Can you reorg this imports how we do in Mimir? stdlib, 3rd party, local packages

Reply:
I opened the issue https://github.com/grafana/rollout-operator/issues/120 since this has shown up in other PRs as well.

Reply:
Fixed in b5eae5a

Reply:
...and I've opened https://github.com/grafana/rollout-operator/pull/121 to fix #120.